crates/api_models/src/payments.rs

```diff
@@ -709,8 +709,8 @@ pub struct AddressDetails {
     pub city: Option<String>,
 
     /// The two-letter ISO country code for the address
-    #[schema(value_type = Option<Country>, max_length = 2, min_length = 2, example = "US")]
-    pub country: Option<api_enums::CountryCode>,
+    #[schema(value_type = Option<CountryAlpha2>, max_length = 2, min_length = 2, example = "US")]
```

Comment:
the min and max length can be removed from this, as this has been converted to enum type